osv/models.py

```diff
@@ -793,7 +793,7 @@ class SourceRepository(ndb.Model):
   bucket = ndb.StringProperty()
   # Vulnerability data not under this path is ignored by the importer.
   directory_path = ndb.StringProperty()
-  # Last synced hash for SourceRepositoryType.GIT.
+  # Last synced hash for SourceRepositoryType.GIT & REST.
```

Comment:
Where is `last_synced_hash` used for REST?

Reply:
Ah maybe it's not used. Got it confused with the sha256 val. Removing.